content/en/docs/concepts/policy/pod-security-standards.md

```diff
@@ -0,0 +1,323 @@
+---
+reviewers:
+- tallclair
+title: Pod Security Standards
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+Security settings for pods are typically applied by using [security
+contexts](../../tasks/configure-pod-container/security-context/). Security Contexts allows for the
```

Comment:
I seem to be missing an extra level of `../` on all the relative links. I'll add it tomorrow.

Reply:
Ah, thanks. I hadn't seen the style guide, and assumed